services/business/istio_validations.go

```diff
@@ -106,15 +107,26 @@ func (in *IstioValidationsService) GetIstioObjectValidations(namespace string, o
 		}
 	}
 
+	// Identical to above, but since k8s layer has both (namespace, serviceentry) and (namespace) queries, we need two different functions
+	fetchNoEntry := func(rValue *[]kubernetes.IstioObject, namespace string, fetcher func(string) ([]kubernetes.IstioObject, error), errChan chan error) {
```

Comment:
Yeah.
I think that we have some duplication in the kubernetes packages and some methods can be joined to unified parameters, but yes, this is correct.